<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarWindow</class>
 <widget class="QDialog" name="CalendarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_calendar">
     <property name="title">
      <string>Обзор</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Найти слоты</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_dates">
     <property name="title">
      <string>Даты</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Настоящая дата</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Начальная дата</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Конечная дата</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="btn_searchRange">
        <property name="text">
         <string>Поиск по диапазону</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="dt_start"/>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="dt_now"/>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="dt_end"/>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="onButtonSend">
        <property name="text">
         <string>Передача данных</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
